src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/caseworker/PaperResponseAdmissionCallbackHandlerTest.java

```diff
@@ -0,0 +1,366 @@
+import uk.gov.hmcts.cmc.claimstore.services.ccd.legaladvisor.DocAssemblyTemplateBodyMapper;
+import uk.gov.hmcts.cmc.claimstore.services.document.DocumentManagementService;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.DefendantResponseNotificationService;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.fixtures.SampleUserDetails;
+import uk.gov.hmcts.cmc.claimstore.utils.CaseDetailsConverter;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.utils.LocalDateTimeFactory;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.docassembly.domain.DocAssemblyResponse;
+import uk.gov.hmcts.reform.document.domain.Document;
+
+import java.time.Clock;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.time.ZoneOffset;
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.cmc.ccd.domain.CCDClaimDocumentType.GENERAL_LETTER;
+import static uk.gov.hmcts.cmc.ccd.domain.CCDPartyType.INDIVIDUAL;
+import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.PAPER_RESPONSE_ADMISSION;
+import static uk.gov.hmcts.cmc.ccd.domain.defendant.CCDResponseType.FULL_ADMISSION;
+import static uk.gov.hmcts.cmc.ccd.domain.defendant.CCDResponseType.PART_ADMISSION;
+import static uk.gov.hmcts.cmc.claimstore.services.ccd.Role.CASEWORKER;
+
+@ExtendWith(MockitoExtension.class)
+@DisplayName("Paper Response Admission handler")
+public class PaperResponseAdmissionCallbackHandlerTest {
+
+    private static final String AUTHORISATION = "Bearer: aaaa";
+    private static final String DOC_URL = "http://success.test";
+    private static final String DOC_URL_BINARY = "http://success.test/binary";
+    private PaperResponseAdmissionCallbackHandler handler;
+    private CallbackParams callbackParams;
+    @Mock
+    private DefendantResponseNotificationService defendantResponseNotificationService;
+    @Mock
+    private CaseMapper caseMapper;
+    @Mock
+    private CaseDetailsConverter caseDetailsConverter;
+    @Mock
+    private DocAssemblyService docAssemblyService;
+    @Mock
+    private DocAssemblyResponse docAssemblyResponse;
+    @Mock
+    private DocumentManagementService documentManagementService;
+    @Mock
+    private Clock clock;
+    @Mock
+    private UserService userService;
+    @Mock
+    private DocAssemblyTemplateBodyMapper docAssemblyTemplateBodyMapper;
+    private UserDetails userDetails;
+
+    @BeforeEach
+    public void setUp() {
```

Comment:
Access should be default for Junit 5 tests
```suggestion
   void setUp() {
```